pkg/fanal/artifact/remote/git.go

```diff
@@ -34,12 +35,16 @@ func NewArtifact(rawurl string, c cache.ArtifactCache, artifactOpt artifact.Opti
 	if err != nil {
 		return nil, cleanup, err
 	}
+	println(artifactOpt.FetchBranch)
+	println(artifactOpt.FetchCommit)
+	println(artifactOpt.FetchTag)
+
+	println(tmpDir)
```

Comment:
debug code?

Reply:
Sorry my bad, forgot to remove the debugs here.